focuspoints.lrdevplugin/PentaxDelegates.lua

```diff
@@ -176,7 +180,7 @@ end
 
 function PentaxDelegates.fixCenter(points)
   for k,v in pairs(points) do
-    if v == 'Center (vertical)' or v == 'Center (horizontal)' then
+    if v == 'Center (vertical)' or v == 'Center (horizontal)' or v == 'Fixed Center' then
```

Comment:
this looks like it could contain a lot of possible combination. It might be hard to keep up with them all. Would it be possible to write a regex to make it more flexible?